### PR TITLE
update docs to incorporate multi chain config

### LINKED ISSUE #42: Update `scaffold.config` `targetNetwork` to be an array of chains
### Description : 

We recently merged https://github.com/scaffold-eth/scaffold-eth-2/pull/615 and it would be great if we reflect same on docs 🙌

---

Also maybe adding small note at https://docs.scaffoldeth.io/hooks/ and also update `externalContract.ts` example to have same contracts name because of this : 

> So currently I see we are assuming that the same contracts are present on others chains as  `scaffoldConfig.targetNetworks[0].id`. 
> 
> Like if for eg: `31337: YourContract`  & `1115511 : YourContract, DAI`
> 1. We are will only give ts autocompletion for `YourContract` and `DAI` is not present in autocompleteions. 
> 2.  We are not showing `DAI` contract on debug page if the user switches the chain to `sepolia`. 

from this comment : https://github.com/scaffold-eth/scaffold-eth-2/pull/615

### PR DESCRIPTION
**Pull request by technophile-04**
Fixes #42

### CODE REVIEW

File: docs/hooks/hooks.md [hunk 1]
```diff
@@ -28,9 +28,15 @@ const externalContracts = {
       address: "0x...",
       abi: [...],
     },
-    USDT: {
+    WETH: {
       address: "0x...",
       abi: [...],
     },
```
Comment (Pabl0cks) [inline docs/hooks/hooks.md:34]: I think in the example we had here there were missing closing brace for each chainId. Afraid the syntax error was in the SE-2 code example too, will create a PR.

Here is the code with the missing closing braces:
```tsx
const externalContracts = {
  1: {
    DAI: {
      address: "0x...",
      abi: [...],
    },
    WETH: {
      address: "0x...",
      abi: [...],
    },
  },
  5: {
    DAI: {
      address: "0x...",
      abi: [...],
    },
    WETH: {
      address: "0x...",
      abi: [...],
    },
  },
} as const;
```

**Review by Pabl0cks — APPROVED** [review]
Looking great to me! 🙌 Just added a comment in the `externalContracts` example object syntax, which was not related with this PR.

File: docs/hooks/hooks.md (hunk 1 above)
Comment (technophile-04) [reply docs/hooks/hooks.md:35]: Tysm !! updated at [568917e](https://github.com/scaffold-eth/se2-docs/pull/45/commits/568917ebe40c883c76678307c8f28b1c3c70d5d5) 🙌

**Comment by Pabl0cks** [comment]
I think we can merge it, and iterate if someone finds it difficult to config their multi chain SE-2 build. 

TYSM @technophile-04, great job!!

**Pull request merged**